lib/virt_disk/block_file.rb

```diff
@@ -1,9 +1,10 @@
 module VirtDisk
   class BlockFile
-    attr_accessor :path
+    attr_accessor :path, :offset
 
-    def initialize(path)
+    def initialize(path, offset=0)
```

Comment:
@movitto Why do you need an offset? All offsets should be handled by the upper level code. Calls down to this layer should be absolute block addresses.